retrofit/src/main/java/retrofit2/Platform.java

```diff
@@ -32,12 +32,14 @@ static Platform get() {
   }
 
   private static Platform findPlatform() {
-    try {
-      Class.forName("android.os.Build");
-      if (Build.VERSION.SDK_INT != 0) {
-        return new Android();
+    if ("Dalvik".equals(System.getProperty("java.vm.name"))) {
```

Comment:
True. It also affects our ability to test this behavior since the Android classes are on the classpath but aren't actually usable.